src/main/java/org/prebid/server/bidder/yieldlab/YieldlabBidder.java

```diff
@@ -231,6 +244,60 @@ private static String getConsentParameter(User user) {
         return ObjectUtils.defaultIfNull(consent, "");
     }
 
+    private static Map<String, String> extractDsaRequestParamsFromBidRequest(BidRequest request) {
+        return Optional.ofNullable(request.getRegs())
+            .map(Regs::getExt)
+            .map(ExtRegs::getDsa)
+            .map(YieldlabBidder::extractDsaRequestParamsFromDsaRegsExtension)
+            .orElseGet(Map::of);
+    }
+
+    private static Map<String, String> extractDsaRequestParamsFromDsaRegsExtension(final ExtRegsDsa dsa) {
+        final HashMap<String, String> dsaRequestParams = new HashMap<>();
```

Comment:
just Map is enough here `final Map<String, String> dsaRequestParams = new HashMap<>();`